.travis.yml

```diff
@@ -17,6 +17,7 @@ sudo: required
 
 branches:
   only:
+    - imports
```

Comment:
By pushing to a branch `issueXYZ`, you can forego the addition of your branch to `.travis.yml`.  See below
```
- /^issue.*/
```

Reply:
OK.